codemagic.yaml

```diff
@@ -171,13 +211,86 @@ workflows:
     publishing:
       email:
         recipients: *debugEmails
+  release-generated-app-builds:
```

Comment:
To be honest - not sure if we need to have this defined. Because realistically we will not make releases here anyways. And this does not trigger manually. 
If this is only here because it runs "stepGeneratedAppBuildRelease" instead of "stepGeneratedAppBuildDebug" and that has more checks than the latter then maybe easier to delete this flow and run "stepGeneratedAppBuildRelease" in the dev builds instead. The something at least runs it regularly
Just a thought.

Reply:
Good poinrt, added `stepGeneratedAppBuildRelease` step to dev build and removed release workflow